mongo/collection.go

```diff
@@ -1304,6 +1305,7 @@ func (coll *Collection) find(ctx context.Context, filter interface{},
 
 	f, err := marshal(filter, coll.bsonOpts, coll.registry)
 	if err != nil {
+		fmt.Println(err)
```

Comment:
Remove debugging `Println`.
```suggestion
```

P.S. Looks like this is in `v1` also. I'll open a PR to remove that line in `v1`.

Reply:
PR for v1: https://github.com/mongodb/mongo-go-driver/pull/1722